### PR TITLE
Removed passing xfail relating to bug 1149187

### PR DESCRIPTION
**Pull request by justinpotts**

### CODE REVIEW

File: tests/test_search.py [hunk 1]
```diff
@@ -60,8 +60,6 @@ def test_search_for_valid_signature(self, mozwebqa):
         Assert.true(result.are_search_results_found)
 
     @pytest.mark.nondestructive
```
Comment (m8ttyB) [inline tests/test_search.py]: Not required but I like that you explicitly mark the test as `nondestructive`.

Comment (justinpotts) [inline tests/test_search.py]: Heh, it was already there. But thanks! :)

**Comment by davehunt** [comment]
r+ can you sqaush the commits before merging?

**Comment by bobsilverberg** [comment]
The commits appear to be squashed, and this test is still xpassing, so I am going to merge this. Thanks for the PR @justinpotts, and sorry for the (massive) delay.

**Pull request merged**